### PR TITLE
osd: pass tcmalloc env through to daemon

### PR DESCRIPTION
**Pull request by liewegas**
Untested!

### CODE REVIEW

File: src/upstart/ceph-osd.conf [hunk 1]
```diff
@@ -22,4 +22,8 @@ instance ${cluster:-ceph}/$id
 export cluster
 export id
 
-exec /usr/bin/ceph-osd --cluster="${cluster:-ceph}" -i "$id" -f --setuser ceph --setgroup ceph
+script
+    . /etc/default/ceph
```
Comment (ktdreyer) [inline src/upstart/ceph-osd.conf]: We should check for the file before sourcing it. The following seems to work:

`test -f /etc/default/ceph && . /etc/default/ceph`

**Comment by ktdreyer** [comment]
other than the thing about checking `/etc/default/ceph` before sourcing, LGTM

File: src/upstart/ceph-osd.conf (hunk 1 above)
Comment (liewegas) [inline src/upstart/ceph-osd.conf]: fixed upstart

**Comment by ktdreyer** [comment]
LGTM!

`Signed-off-by: Ken Dreyer <kdreyer@redhat.com>`

**Pull request merged**